stdlib/public/Concurrency/TaskSleepDuration.swift

```diff
@@ -117,6 +117,7 @@ extension Task where Success == Never, Failure == Never {
     }
   }
 
+  #if !$Embedded
```

Comment:
@al45tair git blame shows that you added some of the checks for `ContinuousClock`, could you take a look and tell me if this is the right thing to do here as well or not? All of the `sleep` methods declared below use `clock: C = ContinuousClock()` which is not available in embedded.

Reply:
Thanks! I'll remove that check.